geoportal/geoportailv3_geoportal/static-ngeo/js/LayerPermalinkManager.js

```diff
@@ -622,12 +622,12 @@ exports.prototype.initBgLayers_ = function() {
       stateBgLayerOpacity = this.ngeoLocation_.getParam('bgOpacity');
     }
     var hasBgLayerInUrl = (this.ngeoLocation_.getParam('bgLayer') !== undefined);
-    if (mapId === undefined || hasBgLayerInUrl) {
```

Comment:
Do we also handle the `mapId === undefined` case in v4? Is this related to mymaps?

Reply:
Thanks for noticing this!
I have added this use case in the default background id selection.